<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenericTableDialog</class>
 <widget class="QWidget" name="BusinessObjectList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>847</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="table1">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>40</y>
     <width>1001</width>
     <height>815</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>690</width>
     <height>0</height>
    </size>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_debug">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>741</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Debug label</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>160</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Generic Button</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTableName">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>400</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Generic Table View 1.0</string>
   </property>
  </widget>
  <zorder>label_debug</zorder>
  <zorder>table1</zorder>
  <zorder>btn1</zorder>
  <zorder>lblTableName</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
